pwndbg/heap/ptmalloc.py

```diff
@@ -697,6 +703,9 @@ def libc_has_debug_syms(self):
 
 
 class DebugSymsHeap(Heap):
+    def can_be_resolved(self):
+        return self.libc_has_debug_syms()
```

Comment:
Would this work?: `can_be_resolved = Heap.libc_has_debug_syms`

Reply:
Yep, it works. I'll change it to your suggestions.